### PR TITLE
removed use of global namepace

### PR DESCRIPTION
**Pull request by chriskuech**
Replaced all uses of $global in the module and tests.  Normalized CRLF line endings in a failing test.

### CODE REVIEW

File: Polaris.psm1 [hunk 1]
```diff
@@ -572,13 +572,22 @@ function Use-JsonBodyParserMiddleware {
     New-RouteMiddleware -Name JsonBodyParser -ScriptBlock $JsonBodyParserMiddlerware
 }
 
+
+function Get-Polaris {
```
Comment (tylerl0706) [inline Polaris.psm1]: Can you add some Help comments?

File: Polaris.psm1 [hunk 2]
```diff
@@ -572,13 +572,22 @@ function Use-JsonBodyParserMiddleware {
...
+    return $script:Polaris
+}
+
+function Clear-Polaris {
```
Comment (tylerl0706) [inline Polaris.psm1]: Can you add some Help comments?

**Comment by chriskuech** [comment]
Just added some :)

**Comment by tylerl0706** [comment]
Thanks for updating those! I just merged in #77 and #78 (Thanks @TimCurwick and @doug-reimer!) there were some pretty big changes and some added `$global:polaris` references to  some test files.

If you have the time, could you address these? Otherwise, I can address it sometime next week when I'm back from vacation. Thanks for contributing!

**Comment by tylerl0706** [comment]
Merged a modified version of your changes here: https://github.com/PowerShell/Polaris/pull/83

Your commits stayed in :) thanks for this!

**Pull request closed without merging**